### ISSUE TITLE
Removing Title Attributes

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4442]
WP Core is moving away from using title attributes in links. It was also [removed from link modal](http://wptavern.com/how-to-restore-the-link-title-attribute-removed-in-wordpress-4-2). In comments there are more info why. 

This is not a big `a11y` issue but could be good to follow this standard also. Title attribute is for example in order detail page in links `View Customer Details  |  New Customer`.

**Comment by pippinsplugins** [comment]
:+1: Let's get them all remvoed in 2.6

**Comment by topher1kenobe** [comment]
There are 36 title tags in anchor tags in EDD core.  Do we want to remove all of them?

There are 9 title tags in other tags like input, textarea, etc.  Do we want to remove those?

**Comment by samikeijonen** [comment]
For anchor links I'd say all. I don't see a reason to leave them somewhere.

For inputs and textareas I need to double check. At the moment I'd vote for removal.

**Comment by pippinsplugins** [comment]
I would think all of them.

**Comment by samikeijonen** [comment]
And my PR #4448 I counted 35. @topher1kenobe can you locate that missing one?

**Comment by ramiy** [comment]
For input and textarea tags, you should replace the title attributes with lable tags.

**Comment by pippinsplugins** [comment]
I found a few remaining title attributes.
